### DISCUSSION TITLE
Gerätebezeichnungen

### DISCUSSION THREAD

**Discussion by msinn** [SukramJ/aiohomematic#1288] [General]
Im Web Interface der CCU gibt es auf der Seite **Geräte** eine Spalte **Bezeichnung**, in der Informationen wie folgende stehen: "Homematic IP Lichtsensor - außen" oder "Homematic IP Fernbedienung, 8 Kanal".

Gibt es in hahomematic eine Möglichkeit auf diese Information zuzugreifen?

**Comment by SukramJ** [comment]
> Gibt es in hahomematic eine Möglichkeit auf diese Information zuzugreifen? 

Ist mir nicht bekannt.
